src/WinRT.Runtime2/InteropServices/ExceptionHelpers.cs

```diff
@@ -0,0 +1,451 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using static System.Runtime.InteropServices.ComWrappers;
+
+namespace WindowsRuntime.InteropServices;
+
+#pragma warning disable IDE0051 // TODO
+
+internal static unsafe class ExceptionHelpers
+{
+    internal const int COR_E_OBJECTDISPOSED = unchecked((int)0x80131622);
```

Comment:
Move all of these to `WellKnownErrorCodes.tt`. Also add a comment if available, you can find some of them e.g. you can find many by looking up via https://james.darpinian.com/decoder/ or [here](https://learn.microsoft.com/en-us/windows/win32/seccrypto/common-hresult-values).